src/main/java/org/jnode/fs/xfs/XfsFileSystem.java

```diff
@@ -48,6 +55,19 @@ public XfsFileSystem(Device device, FileSystemType<? extends FileSystem<XfsEntry
         super(device, true, type);
     }
 
+    public static String HexToAscii(String hexString) {
```

Comment:
This function is generic enough to move to either `org.jnode.fs.util.FSUtils` or `or.jnode.fs.FileSystemTestUtils`. It will probably be only used in test so it can be moved to the second class (unless there's a reason to print hex values in an exception in the production code).

Reply:
Looking at how this is being used: this can probably be done away with. The test in `XfsFileSystemTypeTest` may be able to treat the "magic" as ASCII directly or just do a comparison against `Superblock.XFS_SUPER_MAGIC`.

Reply:
function removed. was not necessary.